Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/ListBetaTestersCommand.swift

```diff
@@ -13,98 +13,102 @@ struct ListBetaTestersCommand: CommonParsableCommand {
+        help: ArgumentHelp(
+            """
+            An invite type that indicates if a beta tester was invited by an email invite or used a TestFlight public link to join a beta test. \n
+            Possible values \(BetaInviteType.allCases).
+            """,
+            valueName: "invite-type"
+        )
+    ) var filterInviteType: BetaInviteType?
+
+    @Option(
+        parsing: .upToNextOption,
+        help: ArgumentHelp(
+            "Filter by app AppStore ID. eg. 432156789",
+            discussion: "This option is mutually exclusive with --filter-bundle-ids.",
+            valueName: "app-id"
+        )
+    ) var filterAppIds: [String]
+
+    @Option(
+        parsing: .upToNextOption,
+        help: ArgumentHelp(
+            "Filter by app bundle identifier. eg. com.example.App",
+            discussion: "This option is mutually exclusive with --filter-app-ids.",
+            valueName: "bundle-id"
+        )
+    ) var filterBundleIds: [String]
+
+    @Option(
+        parsing: .upToNextOption,
+        help: ArgumentHelp(
+            "TestFlight beta group names.",
+            valueName: "group-name"
+        )
+    ) var filterGroupNames: [String]
+
+    @Option(help: "Number of resources to return. (maximum: 200)")
+    var limit: Int?
+
+    @Option(
+        parsing: .unconditional,
+        help: ArgumentHelp(
+            "Sort the results using the provided key \(ListBetaTesters.Sort.allCases).",
+            discussion: "The `-` prefix indicates descending order."
+        )
+    ) var sort: ListBetaTesters.Sort?
+
+    @Option(help: "Number of included related resources to return.")
+    var relatedResourcesLimit: Int?
+
+    func validate() throws {
+        if !filterAppIds.isEmpty && !filterBundleIds.isEmpty {
+            throw ValidationError("Filtering by both Bundle ID and App ID is not supported!")
         }
-    }
 
-    private enum ListStrategy {
-        case all
-        case listByApp(bundleId: String)
-        case listByGroup(betaGroupName: String)
-        case listByAppAndGroup
-
-        typealias ListOptions = (bundleId: String?, betaGroupName: String?)
-
-        init(options: ListOptions) {
-            switch (options.bundleId, options.betaGroupName) {
-                case let(.some(bundleId), .some(betaGroup)) where !bundleId.isEmpty && !betaGroup.isEmpty:
-                    self = .listByAppAndGroup
-                case let(.some(bundleId), _) where !bundleId.isEmpty:
-                    self = .listByApp(bundleId: bundleId)
-                case let(_, .some(betaGroupName)) where !betaGroupName.isEmpty:
-                    self = .listByGroup(betaGroupName: betaGroupName)
-                case (_, _):
-                    self = .all
-            }
+        if (!filterAppIds.isEmpty || !filterBundleIds.isEmpty) && !filterGroupNames.isEmpty {
+            throw ValidationError("Only one of these relationship filters ('app-id, bundle-id', 'group-name') can be applied.")
```

Comment:
I feel like you should be able to filter by both app and beta group?
Does the API not support it?

Reply:
The API will return Error when input both 😢 
```
{
  "errors": [
    {
      "status": "400",
      "code": "PARAMETER_ERROR.INVALID",
      "title": "A parameter has an invalid value",
      "detail": "Only one relationship filter can be applied.",
      "source": {
        "parameter": "filters"
      }
    }
  ]
}
```